### PR TITLE
did task but can not pass tests..

### PR DESCRIPTION
**Pull request by Pokhvalenkos**
- [DEMO LINK](https://Pokhvalenkos.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Pokhvalenkos.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -1,3 +1,77 @@
+$cell-size: 100px;
+$gap: 1px;
+$digit-size: 0.8 * $cell-size;
+$shift: $cell-size / 2;
+$week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+$cell-count: 7;
```
Comment (SanyaBratashchuk) [inline src/styles/index.scss]: move variables to a separate file

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,3 +1,77 @@
...
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
```
Comment (SanyaBratashchuk) [inline src/styles/index.scss]:
```suggestion
```
use `display: flex` for parent element to center calendar

File: src/styles/index.scss [hunk 3]
```diff
@@ -1,3 +1,77 @@
...
+  width: ($cell-size + $gap) * $cell-count;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    position: relative;
```
Comment (SanyaBratashchuk) [inline src/styles/index.scss]:
```suggestion
 displayL flex;
 justify-content: center;
 align-items: center;
```

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,3 +1,77 @@
...
+
+        position: absolute;
+
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        text-align: center;
```
Comment (SanyaBratashchuk) [inline src/styles/index.scss]:
```suggestion
```

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,77 @@
...
+    @for $count from 1 through 31 {
+      &:nth-child(#{$count}n)::before {
+        content: '#{$count}';
+
+        width: $digit-size;
+        height: $digit-size;
+
+        position: absolute;
+
+        font-family: Arial, Helvetica, sans-serif;
```
Comment (SanyaBratashchuk) [inline src/styles/index.scss]: apply font-family for `html` or `body` selector  
```suggestion
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/styles/index.scss (hunk 4 above)
Comment (anastasiiavorobiova) [reply src/styles/index.scss]: Consider moving this style to a general class instead of generating the same styling in the cycle for every 31 element

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Make sure to deploy your solution

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, your demo link is empty now
image

**Pull request closed without merging**